tests/testthat/test-unnecessary_placeholder_linter.R

```diff
@@ -34,6 +34,6 @@ patrick::with_parameters_test_that(
       unnecessary_placeholder_linter()
     )
   },
-  .test_name = c("forward", "assignment", "tee"),
-  pipe = c("%>%", "%<>%", "%T>%")
+  .test_name = names(pipes(exclude = "|>")),
```

Comment:
Any reason this test doesn't use the `local({ pipes <- ... })` pattern?

Reply:
I do prefer avoiding it, but with the new change it's probably better to switch